unit_test/fb_serializer_test.cc

```diff
@@ -71,7 +71,7 @@ BOOST_AUTO_TEST_CASE(fb_serializer_ranking_event) {
   const timestamp ts;
   const size_t events_count = 10;
   for (size_t i = 0; i < events_count; ++i) {
-    mode = static_cast<learning_mode>(i % 2);
+    mode = static_cast<learning_mode>(i % 3);
```

Comment:
might be good code hygiene to have this 3 as it's own const since it's being referenced multiple times

Reply:
sounds good. fixed.